src/components/api/index.tsx

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+const BASE_URL = 'https://react-gift-mock-api-harugi7.vercel.app/';
```

Comment:
[MAJOR]
```suggestion
const BASE_URL = 'https://react-gift-mock-api-harugi7.vercel.app';
```

BASE_URL의 끝에 슬래시는 fetchData를 호출할때 파라미터로 받는 endpoint 앞에 붙여주는 것이 좋겠어요